lib/MetaCPAN/Document/File.pm

```diff
@@ -427,12 +427,12 @@ ArrayRef of ArrayRefs of offset and length of pod blocks. Example:
 =cut
 
 has pod_lines => (
-    is         => 'ro',
-    required   => 1,
-    isa        => 'ArrayRef',
-    type       => 'integer',
-    lazy_build => 1,
-    index      => 'no',
+    is      => 'ro',
+    isa     => 'ArrayRef',
```

Comment:
I think we should make sure to use our own type library wherever possible and just import the types we're using.

```
use MetaCPAN::Types qw( ArrayRef );

...
isa => ArrayRef,
...
```